saleor/plugins/base_plugin.py

```diff
@@ -132,10 +133,12 @@ def calculate_order_shipping(
         """
         return NotImplemented
 
+    # TODO: Add information about this change to `breaking changes in changelog`
     def calculate_checkout_line_total(
         self,
         checkout_line: "CheckoutLine",
         discounts: List["DiscountInfo"],
+        channel: "Channel",
```

Comment:
What do you thing to pass here `channel_id` instead of channel object?

Reply:
In this case, we can't do it because we pass the channel to the plugin. Some plugin might to require channel slug and others can require channel id.